### ISSUE TITLE
AndroidManifest missing @style/LandingPageStyle

### ISSUE THREAD

**Issue by digitalhobo** [urbanairship/react-native-module#77]
hello

react-native-module/android/src/main/AndroidManifest.xml
is missing `"@style/LandingPageStyle"` on
line 42

```
<activity android:name="com.urbanairship.reactnative.CustomLandingPageActivity"
            android:exported="false"
            android:theme="**@style/LandingPageStyle**">
```

**Comment by idrakimuhamad** [comment]
Yeah I faced this to in #76. For now I've downgraded to 1.3.2

**Comment by rlepinski** [comment]
Yes sorry, 1.4.0 was a bad upgrade with missing commits. I fixed iOS in 1.4.1 yesterday but I made a mistake and did not verify Android. We will release 1.4.2 today.

**Comment by rlepinski** [comment]
Fixed in 1.4.2

**Comment by digitalhobo** [comment]
👍
